product_docs/docs/pgd/5/cli/command_ref/node/get-option.mdx

```diff
@@ -0,0 +1,89 @@
+---
+title: pgd node get-option
+navTitle: Get Option
+---
+
+## Synopsis
+
+The `pgd node get-option` command is used to get node-level options in the EDB Postgres Distributed cluster.
+
+## Syntax
+
+```
+pgd node <NODENAME> get-option [OPTIONS] [OPTION]
```

Comment:
well, looks like we have used `NODENAME` like pattern for resource names in many places. I would prefer the `NODE_NAME` though as it is more readable